lib/workers/repository/onboarding/pr/index.spec.ts

```diff
@@ -63,6 +63,20 @@ describe('workers/repository/onboarding/pr/index', () => {
       ]);
     });
 
+    it('creates PR with footer and header', async () => {
+      await ensureOnboardingPr(
+        {
+          ...config,
+          prHeader: 'And this is a header',
+          prFooter: 'And this is a footer',
```

Comment:
These should include handlebars templating to verify that it's working, instead of being plain strings

Reply:
```suggestion
          prHeader: 'Click [here](https://ci.domain.test/{repository}/startJob) to force a renovate run',
          prFooter: 'Click [here](https://ci.domain.test/{repository}/startJob) to force a renovate run',
```

Eg something like this

Reply:
done in fix: review